plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -451,7 +451,7 @@ def _apply_wire_mesh(
         mesh_vdir=None,
     ):
         """
-        Apply wire meshes that were added to self.mesh_list
+        Apply wire meshes that were added to self.mesh_list.
```

Comment:
I see that you added the full stop (period) but in this case, could it potentially cause confusion? I am thinking of a newish user who may think it is included in the code